src/core_modules/capture-core/components/WidgetEnrollment/MiniMap/MiniMap.types.ts

```diff
@@ -0,0 +1,26 @@
+import { WithStyles } from '@material-ui/core';
+import { dataElementTypes } from '../../../metaData';
+
+export type QueryRefetchFunction = (options?: Record<string, any>) => Promise<any>;
+
+export type MiniMapProps = {
+    coordinates: Array<Array<[number, number]>>;
+    enrollment: any;
+    refetchEnrollment: QueryRefetchFunction;
+    refetchTEI: QueryRefetchFunction;
+    onError?: (message: string) => void;
+    geometryType: typeof dataElementTypes.COORDINATE | typeof dataElementTypes.POLYGON;
+} & WithStyles<typeof styles>;
```

Comment:
Remove `& WithStyles<typeof styles>` and fix the 'Property 'classes' does not exist on type 'OwnProps'' error by using `OwnProps & WithStyles<typeof styles>` on line 30 in src/core_modules/capture-core/components/WidgetEnrollment/MiniMap/MiniMap.component.tsx file